### DISCUSSION TITLE
New names

### DISCUSSION THREAD

**Discussion by AlekSi** [FerretDB/FerretDB#100] [Announcements]
We are happy to announce the new product name, the new company name, and, well, our names.

We decided to rename it to FerretDB because ferrets are cool. "Ferret" also does not share a single letter with "Mongo", so the possibility of confusion and trademark infringement should be minimal. That would also allow the existing [awesome MangoDB project](https://github.com/dcramer/mangodb) to be the only one with that name.

I'm also happy to start contributing under my real name. FerretDB is our main project now, and we are 100% dedicated to making it the best open-source MongoDB®-compatible database.

We also made a legal entity with the same name to employ us. You can read more at https://www.ferretdb.io.

Finally, the community Slack is there! You can register [here](https://join.slack.com/t/ferretdb/shared_invite/zt-zqe9hj8g-ZcMG3~5Cs5u9uuOPnZB8~A). We are going to use for ephemeral short-term discussions; please continue to use GitHub discussions and issues for longer topics.

**Comment by OpenSauce** [comment]
Great news - and a new picture, I never did like the yellow mango logo 👍

**Comment by ayakhont** [comment]
I'll be nostalgic for the cheeky mango =)
